Fixed #30934 -- Included database alias in django.db.backends log messages. (#14584)
django/django: docs/ref/logging.txt

``COMMIT``, and ``ROLLBACK``). Turn on query logging in your database if you
wish to view all database queries.

.. versionchanged:: 4.0

>     Messages now include the ``alias`` where the query was executed.

Reviewer: Maybe:
```suggestion
    The database ``alias`` was added to log messages.
```
